src/physical_dimensions/spatial.jl

```diff
@@ -237,18 +237,18 @@ latitudes(::LonLatGrid, A) = dims(A, Lat).val
 #########################################################################
 export tropics_extratropics
 """
-    tropics_extratropics(A::ClimArray) → tropics, extratropics
-Separate the given array into two arrays: one having latitudes ℓ ∈ [-30, 30], and one
-having all other remaining latitudes.
+    tropics_extratropics(A::ClimArray; lower_lat=30, higher_lat=90) → tropics, extratropics
+Separate the given array into two arrays: one having latitudes ℓ ∈ [-lower_lat, +lower_lat], and one
+having [-higher_lat:-lower_lat, lower_lat:higher_lat]. lower_lat defaults to 30 degrees and higher_lat defaults to 90.

```

Comment:
```suggestion
having [-higher_lat:-lower_lat, lower_lat:higher_lat].
```
Stating the default values is already done in the docstring.